README.md

```diff
@@ -316,7 +299,7 @@ changes in [UPGRADE.md](./UPGRADE.md) and [CHANGELOG.md](./CHANGELOG.md).
 
 ### Command line documentation
 
-Run `kratos -h` or `kratos help`.
+Run <code textrun="run-console-command">kratos -h</code> or <code textrun="run-console-command">kratos help</code>.
```

Comment:
This runs `kratos -h` and `kratos help` in a subshell and checks that it returns exit code 0. This action is built into Text-Runner, documentation is [here](https://github.com/kevgo/text-runner/blob/master/documentation/built-in-actions/run_console_command.md).